app_allocator/classes/criterion.py

```diff
@@ -0,0 +1,24 @@
+from app_allocator.classes.feature import Feature
+
+
+DEFAULT_COUNT = 1
+DEFAULT_WEIGHT = 1
+
+
+class Criterion(object):
+    def __init__(self, name):
+        self.feature = Feature(type=self.type, name=name)
+        self.count = DEFAULT_COUNT
+        self.weight = DEFAULT_WEIGHT
+
+    def name(self):
+        return self.feature.name
+
+    def setup(self, judges, applications):
+        pass
+
+    def add_option(self, option, count=None, weight=None):
+        if count:
```

Comment:
Note that there is a potential gotcha here: this expects `count` and `weight` to be `str`s, not `int`s. If `int` is used, then this will fail in the (admittedly peculiar) case of a zero-count or zero-weight criterion. (since zero is falsey, we'll get the default value of 1 instead of the expected zero)

I think this is fine to leave as is, since currently we're only passing values read in from a csv, but might be worth leaving a comment to avoid future regressions. (ie, someone decides to handle the `int`-ing upstream, or writes something that creates criteria more dynamically rather than via csv)

Reply:
Addressed the issue.